spec/supabase_js_v2.yml

```diff
@@ -333,17 +333,17 @@ pages:
     title: 'setSession()'
     $ref: '@supabase/gotrue-js.GoTrueClient.setSession'
     notes: |
-      - `setSession()` takes in a refresh token and uses it to get a new session.
+      - `setSession()` takes in a refresh token and access token and uses it to get a new session.
```

Comment:
It looks like it's a refresh token OR an access token right?

https://github.com/supabase/gotrue-js/blob/cf0d32c40d67bc1137c603854583a7cec67017e0/src/GoTrueClient.ts#L582

Reply:
I only updated the "Refresh the session" example, as it doesn't work anymore without those changes. The actual setSession() method seems to now take either a `Session` object or an access_token or refresh_token. But the session object can be constructed as `{access_token, refresh_token}`. The changes made seem unclear, but the example I've updated corresponds to the updated usage: 
https://github.com/supabase/gotrue-js/pull/445#issue-1380749251
Where it's access_token and refresh_token. Looking at the code, the `currentSession` param is decomposed into the tokens, so parsing an object that looks like `{refresh_token, access_token}` works. 
See: https://github.com/supabase/gotrue-js/blob/cf0d32c40d67bc1137c603854583a7cec67017e0/src/GoTrueClient.ts#L589

Reply:
@Aerilym @kiwicopple `setSession()` requires both an access token and refresh token now. The current doc is outdated. The Notes section also needs updated, not just the change in this PR.

@hf has PR [#511](https://github.com/supabase/gotrue-js/pull/511) to fix the `Pick` typing, because the documentation tool doesn't understand it; so the Parameters section doesn't render correctly.

Reply:
Oh hail. I'm just noticing the additional changes in the code below. So forget my comments about also needing to update the Notes and Examples sections; not to mention the code above IS the Notes section. I've not been paying attention!

Although I do feel that changing the name of the example may be warranted:
```diff
- - name: Refresh the session
+ - name: Set the session

Reply:
Thanks for that insight, and I agree with changing the name to better reflect the example usage and have made that change. I assume this resolves any issues brought up through his comment but let me know if I've missed anything.